Desktop/exellenteam_python/vingert.py

```diff
@@ -0,0 +1,20 @@
+import random
+import datetime
+
+strdate1=input("enter a date (YYYY-MM-DD)")
```

Comment:
dont add numbers in variable names unless its for a good reason - in here, it makes way more sense to call the variables start_date and end_date - they have different meanings, they are not just two dates

Reply:
change all your variable names to ones with meaning